packages/ui/styles/constants/_sizes.scss

```diff
@@ -26,6 +26,8 @@ $iconStrokeWidthBold: 3px !default;
 
 $iconSizeSmall: $grid * 4 !default;
 $iconSizeNormal: $grid * 6 !default;
+$iconSizeMedium: $grid * 8 !default;
+$iconSizeLarge: $grid * 10 !default;
 $iconSizeXLarge: $grid * 12 !default;
```

Comment:
I am wondering if the number of icon sizes isn't too big now. Can you ask @pawelklasa if this is alright?

Reply:
It is, I forgot to remove `medium`, which was an experiment. Addressed.